packages/synapse-react-client/src/style/base/_core.scss

```diff
@@ -21,6 +21,16 @@
 a:hover {
   cursor: pointer;
 }
+.forcePlotlyDefaultCursor {
+  .js-plotly-plot {
+    .plotly {
+      .drag {
+        cursor: default !important;
```

Comment:
For some reason, even when we tell plot.ly to not be interactive, it changes the cursor as if it still is interactive!  Override via css